vars/mtaBuild.groovy

```diff
@@ -1,16 +1,41 @@
 import com.sap.piper.Utils
 
+import com.sap.piper.ConfigurationLoader
+import com.sap.piper.ConfigurationMerger
+import com.sap.piper.ConfigurationType
+
 
 def call(Map parameters = [:]) {
 
-    handlePipelineStepErrors (stepName: 'mtaBuild', stepParameters: parameters) {
+    def stepName = 'mtaBuild'
+
+    List parameterKeys = [
+        'buildTarget',
+        'mtaJarLocation'
+    ]
+
+    List stepConfigurationKeys = [
+        'mtaJarLocation'
+    ]
+
+    handlePipelineStepErrors (stepName: stepName, stepParameters: parameters) {
+
+        def script = parameters?.script ?: [commonPipelineEnvironment: commonPipelineEnvironment]
+
+        setupCommonPipelineEnvironment script: script
+
+        final Map customStepConfiguration = [:]
+        customStepConfiguration.putAll(ConfigurationLoader.stepConfiguration(script, stepName))
```

Comment:
Is there a reason for first initializing the map (empty) and populating it with the putAll?
In general this decouples the map returned by the ConfigurationLoader from the map being used later on. This would be a benefit in case we would like to avoid write-backs into the configuration. But later in the ```ConfigurationMerger.merge``` we get a new map anyhow. IMO it is for now sufficient to have ```dev customStepConfiguration = ConfigurationLoader.stepConfiguration(./.). In addition we should clarify the expected behaviour wrt write-backs into the configuration, but I this should IMO not be part of that PR.